src/converse-chatview.js

```diff
@@ -196,6 +196,7 @@ converse.plugins.add('converse-chatview', {
                 this.initDebounced();
 
                 this.listenTo(this.model.messages, 'add', this.onMessageAdded);
+                this.listenTo(this.model.messages, 'change:edited', this.onMessageEdited);
```

Comment:
This is for one-on-one chats only, so we need a similar line in the `initialize` method of `_converse.ChatRoomView`.